src/pages/ProductList/ProductList.scss

```
@@ -0,0 +1,144 @@
+.menuPath {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 50px;
+  height: 60px;
+  width: 230px;
+  font-size: 13px;
+  font-weight: 350;
+  color: #878787;
+
+  .menuPathSlash {
+    padding: 0 10px;
+  }
+}
+
+.categoryHeader {
+  display: flex;
+  margin-bottom: 100px;
+
+  .categoryIntroPic {
+    width: 47.5%;
+    height: auto;
+  }
+
+  .categoryIntro {
+    margin: 0.5% 0 0 3%;
+    width: 40%;
+
+    .categoryTitle {
+      margin-bottom: 20px;
+      font-size: 25px;
+    }
+
+    .categoryDescription {
+      margin-bottom: 20px;
+      line-height: 22px;
+      font-size: 15px;
+      font-weight: 350;
+    }
+
+    .subCategoryBox {
+      display: flex;
+      height: 20%;
+      line-height: 25px;
+      font-size: 15px;
+
+      .subCategoryBoxFirst {
+        margin-right: 40px;
+      }
+    }
+
+    .filterFunction {
+      display: flex;
+      justify-content: space-between;
+
+      .filteringResult {
+        padding-top: 20px;
+        font-size: 15px;
+      }
+
+      .filterAndAlign {
+        padding: 10px 30px;
+        background-color: rgb(41, 40, 40);
+        border: 0.1px solid rgb(78, 76, 76);
+        border-radius: 27px;
+        font-size: 15px;
+        font-weight: 330;
+        color: white;
+
+        .fa-sliders-h {
+          padding-right: 20px;
+        }
+      }
+
+      .filterAndAlign:hover {
+        background-color: white;
+        color: black;
+      }
+    }
+  }
+}
+
+.productList {
+  font-size: 13px;
+  position: relative;
+
+  .productListLine {
+    display: flex;
+    justify-content: space-around;
+
+    .eachProduct {
+      width: 20%;
+      height: auto;
+
+      .eachProductPic {
+        width: 100%;
+        height: auto;
+      }
+
+      .eachProductInfo {
+        display: flex;
+        flex-direction: column;
+        position: relative;
+        width: 80%;
+        bottom: 40px;
+
+        .eachProductColor {
+          position: relative;
```

```suggestion
      .eachProductInfo {
        position: relative;
        width: 80%;
        bottom: 40px;

        .eachProductColor {
          position: absolute;
```
- 세세한 css 스타일링 까지는 말씀드리지는 않겠지만 기본적으로 flex에만 의존하시기 보다 태그가 기본적으로 가지고 있는 속성인 display 속성을 잘 활용하시는게 기본입니다. 세로 정렬을 하기 위해서 flex direction column을 사용하실 수도 있지만 block 요소들은 처음부터 세로 정렬되는 특징을 가지고 있습니다.
- 자기 상위 요소를 기준으로 위치를 정하는 것이기 때문에 relative보다는 absolute가 더 맞지 않을까요?